gym_multigrid/core/agent.py

```diff
@@ -1,60 +1,75 @@
 import enum
 import math
 from typing import Type, TypeVar
-
```

Comment:
A line is intentionally inserted to separate Python's standard libs and 3rd party libs